src/apis/comment.ts

```diff
@@ -0,0 +1,24 @@
+import { axiosInterface } from './axios';
+
+type CommentData = {
+  comment: string;
+  postId: string;
+};
+export const sendComment = async ({ comment, postId }: CommentData) => {
```

Comment:
```suggestion
export const createComment = async ({ comment, postId }: CommentData) => {
```
`createComment`는 어떤가요??

Reply:
저도 그것만 제외하고 괜찮은것 같습니다!